cassandra/client.go

```diff
@@ -322,21 +331,23 @@ func addSslOptions(cluster *gocql.ClusterConfig, options *sslOptions) *gocql.Clu
 	return cluster
 }
 
-func initializeSession(cluster *gocql.ClusterConfig) *gocql.Session {
+func initializeSession(cluster *gocql.ClusterConfig, co clientOptions) *gocql.Session {
 	session, err := cluster.CreateSession()
 	if err != nil {
 		log.Fatal(err.Error())
 	}
 
-	if err := session.Query(createKeyspaceCQL).Exec(); err != nil {
-		log.Fatal(err.Error())
+	if co.createKeyspace {
```

Comment:
Here you've got some proof of importance of this flag. 

File `task_w_creating_keyspace.yml` - emulates situation, when this flag isn't needed
File `task_wo_creating_keyspace.yml` - prevents from creating keyspace.

As you can see creating keyspace is killing first task, while second task is up and running.
![screencast_05-22-2017_02-12-33 pm](https://cloud.githubusercontent.com/assets/9694425/26308632/4f26e6fe-3efa-11e7-8161-2e6ca11928fd.gif)